costmanagement-metrics-operator/3.3.1/manifests/costmanagement-metrics-operator.clusterserviceversion.yaml

```diff
@@ -608,8 +612,5 @@ spec:
   minKubeVersion: 1.24.0
   provider:
     name: Red Hat
-  version: 3.3.0
-  relatedImages:
-    - name: koku-metrics-operator
-      image: quay.io/project-koku/koku-metrics-operator@sha256:0c8091f0c04593c19249755bde418cc09e5d30cc55c6cbff2c5b60bdf60dfde9
```

Comment:
Just FYI, in regards to Konflux, this `relatedImages` is the thing that the docs mention. Brew automatically adds this to the CSV, but Konflux does not.

Reply:
👍🏾 okay, sounds good!